frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -50,19 +48,20 @@ function FileManagerMenu:init()
 end
 
 function FileManagerMenu:initGesListener()
-    self.ges_events = {
-        TapShowMenu = {
-            GestureRange:new{
-                ges = "tap",
-                range = Geom:new{
-                    x = 0,
-                    y = 0,
-                    w = Screen:getWidth()*3/4,
-                    h = Screen:getHeight()/4,
-                }
-            }
+    if not Device:isTouchDevice() then return end
+
+    self:registerTouchZones({
```

Comment:
As far as I can tell `SetDimensions` is the file manager's equivalent to `ReaderReady`?